src/thor/bidirectional_astar.cc

```diff
@@ -228,6 +229,7 @@ bool BidirectionalAStar::ExpandForward(GraphReader& graphreader,
 // edge is a not-thru edge that will be pruned.
 //
 inline bool BidirectionalAStar::ExpandForwardInner(GraphReader& graphreader,
+                                                   graph_tile_ptr& potential_tile,
```

Comment:
this one and the ExpandReverseInner dont need to change they already send the cached tile along